install/kubernetes/templates/istio-ingress.yaml.tmpl

```diff
@@ -91,7 +91,7 @@ spec:
       volumes:
       - name: istio-certs
         secret:
-          secretName: istio.default
+          secretName: istio-ingress-service-account
```

Comment:
Should be istio.istio-ingress-service-account

Reply:
The secret name format is istio.$(ServiceAccountName)

Reply:
Thanks for pointing that out. Fixed.